stormforge/README.md

```diff
@@ -0,0 +1,57 @@
+# StormForge Optimize Live
+
+## Overview
+
+[StormForge Optimize Live][1] applies machine learning to your observability metrics to make real-time recommendations to resource requests for any deployment running in Kubernetes.
+
+For access to Optimize Live, please contact your StormForge sales rep directly or through the Datadog marketplace.
+
+**Benefits Include:**
+- Improve resource efficiency
+- Leverage existing observability data
+- Reduce risk of performance issues
+- Fast time to value
+- Deploy recommendations automatically or with approval
+
+## Setup
+
+To set up this integration, you must have a StormForge account along with Datadog API and Application keys.
+
+### Configuration
+
+1. Create a [Datadog API key][2].
+2. Create a [Datadog Application key][3].
+3. Add Datadog API and Application keys to the [StormForge Datadog Integration][4].
+4. Deploy Optimize Live
+5. Set up your Applications within [StormForge][5].
```

Comment:
```suggestion
5. Set up your applications within [StormForge][5].
```

Reply:
We use `Application` as a term within our product. Is the capitalization fair or does it need to be changed?

Reply:
Ah, I didn't know that. Yes, you can keep Application capitalized, in that case.

Reply:
However, in "Datadog application key", "application" should be lower case.